TitleBarCustomization/explainer.md

```diff
@@ -242,6 +274,16 @@ resizeTitleBar();
 window.addEventListener('resize', resizeTitleBar);
 ```
 
+## Security Considerations
+
+Giving sites partial control of the title bar leaves room for developers to spoof content in what was previously a trusted, UA-controlled region. 
+
+Currently in Chromium browsers, `standalone` mode includes a title bar which on initial launch displays the `title` of the webpage on the left, and the origin of the page on the right (followed by the "settings and more" button and the caption controls). After a few seconds, the origin text disappears. 
+
+In RTL configured browsers, this layout is flipped such that the origin text is on the left. This open the caption controls overlay to spoofing the origin if there is insufficient padding between the origin and the right edge of the overlay. For example, the origin "evil.ltd" could be appended with a trusted site "google.com", leading users to believe that the source is trustworthy.  
```

Comment:
Did you mean "evil.tld"?

Reply:
According to the internet, .ltd is a domain for limited liability companies. I was just looking for something that was not an obvious domain since folks might notice someone spoofing if it looks like "evil.com.google.com". My guess was that "evil.ltd.google.com" might be more likely to go unnoticed. So my solution was Evil LLC. 

That said, it doesn't matter what domain is used here, so if there is some reason that I should change it, LMK

Reply:
Gotcha. I was thinking you intended it to be "tld" as in "Top Level Domain" which we often use in docs as a placeholder for any top-level domain. I’m fine either way.

Reply:
830fc38baedb20a68ac4089b10fd8591a7ba8fb4 please fuck me 812